project/make/.dockerinit_gccgo

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+IAMSTATIC="true"
+source "$(dirname "$BASH_SOURCE")/.go-autogen"
+
+EXTLDFLAGS_STATIC_DOCKER="-static -Wl,--no-export-dynamic $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
```

Comment:
EXTLDFLAGS_STATIC_DOCKER is already set in make.sh. Do we really need to reset it again here ?